src/components/Team/index.tsx

```diff
@@ -58,6 +58,12 @@ const TeamView = () => {
   };
 
   const filteredTeams = filterteams(teams);
+
+  const handleDelete = async (team: Team) => {
+    await firebase.deleteTeam(team.id);
+    const teamsAfterDelete = teams.filter(t => t.id !== team.id);
```

Comment:
Don't you think it'd be better to fetch the teams again after deletion instead of just filtering the current list in the view?

Reply:
Yes, I do.